common/istio/profile.yaml

```diff
@@ -29,6 +29,6 @@ spec:
     global:
       configValidation: true
       istioNamespace: istio-system
-      proxy:
-        seccompProfile:
-          type: RuntimeDefault
+      # proxy: # TODO: Uncomment to enable seccompProfile for sidecar proxies
+      #   seccompProfile:
+      #     type: RuntimeDefault
```

Comment:
Why are you disabling that ?

Reply:
Can we leave it enabled such that it automatically kicks in in the next istio upgrade?